docs/style-guide.md

```diff
@@ -0,0 +1,612 @@
+Protractor style guide
+======================
+
+This style guide was originally created by
+[Carmen Popoviciu](https://github.com/CarmenPopoviciu) and 
+[Andres Dominguez](https://github.com/andresdominguez). It is based on Carmen's 
+Protractor 
+[style guide](https://github.com/CarmenPopoviciu/protractor-styleguide) and 
+Google's Protractor style guide.
+
+## Video
+
+Carmen and Andres gave a talk about this style guide at 
+[AngularConnect](http://angularconnect.com/) in London. Here's the video in 
+case you want to watch it.
+
+<a href="http://www.youtube.com/watch?feature=player_embedded&v=-lTGnYwnEuM" 
+  target="_blank"><img src="http://img.youtube.com/vi/-lTGnYwnEuM/0.jpg" 
+  alt="Protractor styleguide @AngularConnect" width="240" height="180" 
+  border="10"/></a>
+
+## Table of contents
+
+* [Generic rules](#generic-rules)
+* [Project structure](#project-structure)
+* [Locator strategies](#locator-strategies)
+* [Page objects](#page-objects)
+* [Test suites](#test-suites)
+
+# Generic rules
+
+### Don't e2e test what’s been unit tested
+
+**Why?**
+* Unit tests are much faster than e2e tests
+* Avoid duplicate tests
+
+### Use one configuration file
```

Comment:
I'm not 100% convinced for this one. I think many setups work fine with multiple config files and sharing code between them via node exports... maybe this could instead just say "share common code between configuration files using node imports"?